README.md

```diff
@@ -1,156 +1,142 @@
-# Ergonode development docker
+# Ergonode Docker
 
-This is only a development solution! Do not use it on production environments!
-
-## the docker uses the following services
+## Ergonode Docker uses the following services
 
  - PostgreSQL 10
  - PHP 7.4
  - NGINX 1.17
  - Docsify 4
  - Nodejs 12.6 
  - RabbitMQ 3.8
- 
-## How the hell should I install it all ?!
+
+## Development use
+
+### Installation
 
 At first you must install Docker and Docker Compose (https://docs.docker.com/compose).
 
-Next, you must clone frontend and backend and docs repositories to ergonode directory:
+Next, you must clone frontend, backend and docs repositories to ergonode directory:
 
-```bash
+```shell script
 git clone git@github.com:ergonode/docker.git ergonode
 cd ergonode
 git clone git@github.com:ergonode/frontend.git
 git clone git@github.com:ergonode/backend.git
 git clone git@github.com:ergonode/docs.git
 ```
 
-
 If you want to test ergonode in multiple directories you need to create the  `.env` file and set
-COMPOSE_PROJECT_NAME env var to some unique value
+`COMPOSE_PROJECT_NAME` env var to some unique value.
 
-If you want't change any environment variable you can optionally  change this in the `.env` file. 
+If you want change any environment variable you can optionally change this in the `.env` file. 
 And all environment variables used by our docker you can find in the `docker-compose.yml` files.
 
-Now you can start start docker by simple command
+Now you can start docker by simple command
 
-```bash
+```shell script
 docker-compose up
 ```
 
-Now you can fill  app database with basic data by using command
-```
-docker-compose exec php bin/phing database:fixture
-```
+### Backend setup
 
-Or fill database with development data with command
-```
-docker-compose exec php bin/phing database:fixture:dev
-```
-
-Enjoy :)
+List of changes that must be made in `ergonode/backend` configuration.
 
-## Ok, but what now?
+#### Messenger DSN
 
+Messenger DSN addresses must be changed to `amqp://ergonode:ergonode@rabbit:5672` in your `.env.local` file.
 
-If you want to view frontend panel just type address from below into your browser
-
+For example:
 ```
-http://localhost
+MESSENGER_TRANSPORT_DSN=amqp://ergonode:ergonode@rabbitmq:5672/%2f/messages
 ```
 
-And to test app you can login as `test@ergonode.com` with password `abcd1234`
-
-If you want to view backend API doc just type address from below into your browser
+#### Mailer DSN
 
-```
-http://localhost:8000/api/doc
-```
+Mailer DSN address must be changed to `smtp://mailhog:1025` in your `.env.local` file.
```

Comment:
docker-compose does not support files like .env.local

Reply:
This sections it's about Ergonode Backend repository configuration. When someone use docker, must change some settings in `.env.local`